go.mod

```diff
@@ -178,6 +179,8 @@ require (
 
 replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	github.com/cosmos/ledger-cosmos-go => github.com/cosmos/ledger-cosmos-go v0.12.2
```

Comment:
I really don't understand why is it needed? Nothing depends on 0.13.0 so it should not be bumped after a go mod tidy and the replacement should not be needed.